tar_scm.service

```diff
@@ -22,12 +22,12 @@
   <param name="versionformat">
     <description>
       Auto-generate version from checked out source using this format
-      string.  This parameter is used if the 'version' parameter is not
+      string. This parameter is used if the 'version' parameter is not
```

Comment:
IMHO it's better to standardize on double-spaces to end sentences, because a) it's visually clearer and b) it allows text editors to provide functionality for navigating forwards and backwards one sentence at a time.